setup.py

```diff
@@ -57,7 +57,8 @@
                   'mock',
```

Comment:
I think `mock` can be removed from the testing dependencies. Python >= 3.3 includes it in the standard library with `unittest`.

Reply:
Good catch, I missed that one previously.